managed/src/main/java/com/yugabyte/yw/controllers/CustomerTaskController.java

```diff
@@ -132,82 +121,37 @@
           taskList.add(taskData);
           taskListMap.put(task.getTargetUUID(), taskList);
         }
-      } catch (RuntimeException e) {
-        LOG.error(
-            "Error fetching Task Progress for "
-                + task.getTaskUUID()
-                + ", TaskInfo with that taskUUID not found");
       }
```

Comment:
I think what this was doing is that it will continue processing even if one task has some data corruption. 
New code is not doing that. 

I think you can safely reduce the try catch block to just the `else` part though.

Reply:
yeah sure.